pkg/private/tar/BUILD

```diff
@@ -0,0 +1,56 @@
+# Copyright 2021 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""rules_pkg internal code.
+
+All interfaces are subject to change at any time.
+"""
+
+load("@rules_python//python:defs.bzl", "py_binary")
+
+licenses(["notice"])
+
+filegroup(
+    name = "standard_package",
+    srcs = [
+        "BUILD",
+    ] + glob([
+        "*.bzl",
+        "*.py",
+    ]),
+    visibility = ["//distro:__pkg__"],
+)
+
+exports_files(
```

Comment:
A style I've found to work well is to have things like `exports_files` be at the top of the file, followed by any other rules.  Makes the special nature of them a little more obvious.

Nothing wrong with this, though.

Reply:
The fact that these have to be exported at all, just to please bzl_library is a failure.